src/test/java/com/wikia/webdriver/common/dataprovider/TrackingOptInDataProvider.java

```diff
@@ -6,16 +6,15 @@
 
 public class TrackingOptInDataProvider {
 
-  private static final String
-      ADS_KIKIMORA_PATTERN =
-      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*";
+  private static final String ADS_KIKIMORA_OPT_IN_ACCEPTED_PATTERN =
+      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*u=0.*";
+  private static final String ADS_KIKIMORA_OPT_IN_REJECTED_PATTERN =
+      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*u=-1.*";
   private static final String ADS_KRUX_PATTERN = "https?://.*cdn\\.krxd\\.net.*";
-  private static final String
-      ADS_NETZ_ATHLETEN_PATTERN =
+  private static final String ADS_NETZ_ATHLETEN_PATTERN =
       "http?://.*adadapter\\.netzathleten-media\\.de/.*/naMediaAd\\.js.*";
   private static final String ADS_MOAT_PATTERN = "https?://.*pixel\\.gif.*";
-  private static final String
-      ADS_PREBID_PATTERN =
+  private static final String ADS_PREBID_PATTERN =
       "http?://.*gads\\.pubmatic\\.com/AdServer/AdCallAggregator.*";
   private static final String ADS_A9_PATTERN = "http?://.*amazon-adsystem\\.com/.*/apstag\\.js.*";
   private static final String ADS_COMSCORE_PATTERN = "http?://.*scorecardresearch.*";
```

Comment:
This pattern will match `scorecardresearch.com` as well as `somethingelsecorecardresearch.com` or `somethingelsescorecardresearchevenmoresomethingelse.net`. Consider it making more strict and just look for calls to Comescore's domain: `scorecardresearch.com`.